pages/01.metakube/01.release-notes/default.en.md

```diff
@@ -10,6 +10,15 @@ page-toc:
 
 ### September 2020
 
+
+#### 2020-10-15
+
+* New clusters get created with etcd launcher.
+* Add tolerations to cinder CSI plugin to be able to mount volumes in tainted nodes.
+* Updated seed infrastructure nodes to Ubuntu 20.04 LTS.
+* Add validation for taints when creating and updating node deployments using the Web UI.
+* Fix kubeconfig API endpoint.
+
```

Comment:
We fixed the kubelet restart issue for new or updated nodes too

Reply:
I would still write that it is only for new/updated ones because older nodes can still have this problem